tracee-ebpf/tracee/filters.go

```diff
@@ -589,3 +591,82 @@ func (filter *ProcessTreeFilter) Set(bpfModule *bpf.Module) error {
 
 	return nil
 }
+
+type ContIDFilter struct {
+	Equal    []string
+	NotEqual []string
+	Enabled  bool
+}
+
+func (filter *ContIDFilter) Parse(operatorAndValues string) error {
+	filter.Enabled = true
+
+	strFilter := &StringFilter{
+		Equal:    []string{},
+		NotEqual: []string{},
+	}
+
+	// Treat operatorAndValues as a string filter to avoid code duplication
+	err := strFilter.Parse(operatorAndValues)
+	if err != nil {
+		return err
+	}
+
+	filter.Equal = strFilter.Equal
+	filter.NotEqual = strFilter.NotEqual
+
+	return nil
+}
+
+func (filter *ContIDFilter) Set(bpfModule *bpf.Module, conts *containers.Containers, filterMapName string, configFilter bpfConfig) error {
+	if !filter.Enabled {
+		return nil
+	}
+
+	filterEqualU32 := uint32(filterEqual) // const need local var for bpfMap.Update()
+	filterNotEqualU32 := uint32(filterNotEqual)
+
+	filterMap, err := bpfModule.GetMap(filterMapName)
+	if err != nil {
+		return err
+	}
+
+	for i := 0; i < len(filter.Equal); i++ {
+		if len(filter.Equal[i]) < 5 {
```

Comment:
I wanted to avoid checking for containers starting with the same prefix, so I chose a minimal 5 chars prefix to avoid collisions. I can add it as a constant. Other suggestion?

Reply:
Oh, you mean like the "min" id to avoid collisions during the check. I see. No suggestion, was trying to understand!